# Ebola_Virus_OmicsLogic
Topic: Multiple Sequence Alignment and Phylogeny Analysis of selected strains of Ebola virus.

To understand whether genomes are different, we can align them to a reference (pairwise alignment) or compare multiple genomes to each other using multiple sequence alignment (MSA). Multiple Sequence Alignment is based on the assumption that genomes vary because of a natural process of mutation accumulation. The phylogenetic tree specifies a topology, length of branches, root positions, and their shapes. This project aims to determine which geological regionâ€™s samples have the closest evolutionary relationship with the reference genome. In this project I studied the evolutionary distance between different Ebola virus strains with the help of a sequence repository called NCBI Virus. I search by virus the Ebolavirus, taxid: 186536. I used MEGA 11 software to perform Multiple Sequence Alignment of the selected samples against the consensus reference genome. The resulting aligned sequence was downloaded in MEGA format. The aligned sequences were then used to construct a phylogeny tree. From this project, I understand that samples with accession numbers: KC242795, KC242797, and KC242798 (all from Gabon) have the closest evolutionary relationship with the reference genome.
